doc/20-advanced/10-repositories.md

```diff
@@ -0,0 +1,103 @@
+
+## `force-data-dir`
+
+Using `image-builder` with the `force-data-dir` argument allows for overriding the built-in repositories. Repositories passed through `force-data-dir` are only used during the build of an artifact, they are not configured or available on the artifact after build.
+
+The expected layout of the data directory passed is as follows:
+
+```
+data
+└── repositories
+    ├── rhel-10.0.json
+    └── rhel-10.1.json
+```
+
+The `.json` files contain the repositories that are used for each distribution and must match one of the distributions in the definitions. The format of these repository files is:
+
+```json
+{
+    "x86_64": [
+        {
+            "name": "BaseOS",
+            "baseurl": "https://some/base/url",
+            "gpgkey": "----BEGIN PGP PUBLIC KEY BLOCK-----\n...\n-----END PGP PUBLIC KEY BLOCK-----\n",
+            "check_gpg": true
+        },
+        { ... },
+        { ... }
+    ],
+    "aarch64": [
+    ]
+}
+```
+
+When `image-builder` is used with `force-data-dir` only the repositories inside the passed data directory are available. This implies that only distributions which have repositories defined are available. In the following command we have the contents of the above example(s) in our `./data` directory.
+
+```shell
+$ image-builder --force-data-dir=./data list
+rhel-10.0 type:ami arch:x86_64
+rhel-10.0 type:azure-cvm arch:x86_64
+rhel-10.0 type:azure-rhui arch:x86_64
+rhel-10.0 type:azure-sap-rhui arch:x86_64
+rhel-10.0 type:azure-sapapps-rhui arch:x86_64
+rhel-10.0 type:ec2 arch:x86_64
+rhel-10.0 type:ec2-ha arch:x86_64
+rhel-10.0 type:ec2-sap arch:x86_64
+# ...
+```
+
+The list only contains `rhel-10.0` and `rhel-10.1` image types as available.
+
+For more information on the format and available options see the [managing repositories](https://osbuild.org/docs/on-premises/installation/managing-repositories/) page.
+
+## `force-repo` / `extra-repo`
+
+It is also possible to override repositories directly from the command line. This offers fewer options to configure repositories. When repositories are passed through `force-repo` or `extra-repo` their contents are not verified. Repositories passed through `force-repo` or `extra-repo` are only used during the build of an artifact, they are not configured or available on the artifact after build.
+
+The following command will build a `minimal-raw-xz` image for Fedora 43 using *only* the repository passed by `--force-repo`. This means that whichever repository is passed must contain all packages necessary.
+
+```shell
+$ sudo image-builder build --distro fedora-43 --force-repo https://some/base/url minimal-raw-xz
+```
+
+`force-repo` can be passed multiple times. When this is done all `force-repo` repositories are used but no built in repositories.
```

Comment:
The word "passed" is used a lot in this section.  Let's reduce the number a bit and instead say "can be used multiple times" (or maybe "specified", if you prefer).

Reply:
I've replaced some with "given", and some with "can be used". Large reduction in the use of `passed` :)